src/main/java/uk/gov/di/ipv/helpers/ResponseBodyHelper.java

```diff
@@ -18,4 +18,5 @@
 
         return query_pairs;
     }
+
```

Comment:
Please avoid committing null. changes without explaination.

Reply:
@SunilDGDS we can actually get rid of `RequestHelper` now as we're not dealing with request parameters

Reply:
extra line removed